synapse/module_api/__init__.py

```diff
@@ -1563,6 +1564,29 @@ async def get_monthly_active_users_by_service(
             start_timestamp, end_timestamp
         )
 
+    async def get_canonical_room_alias(self, room_id: RoomID) -> Optional[RoomAlias]:
+        """
+        Retrieve the given room's current canonical alias.
+
+        A room may declare an alias as "canonical", meaning that it is the
+        preferred alias to use when referring to the room. This function
+        retrieves that alias from the room's state.
+
+        Added in Synapse v1.84.0.
```

Comment:
Sadly I think this needs to be updated again.

```suggestion
        Added in Synapse v1.86.0.
```